ext/tags.go

```diff
@@ -19,30 +19,32 @@ var (
 	//////////////////////////////////////////////////////////////////////
 
 	// SpanKind hints at relationship between spans, e.g. client/server
-	SpanKind = spanKindTag("span.kind")
+	SpanKind = spanKindTagName("span.kind")
 
 	// SpanKindRPCClient marks a span representing the client-side of an RPC
 	// or other remote call
-	SpanKindRPCClient = SpanKindEnum("client")
+	SpanKindRPCClientEnum = SpanKindEnum("client")
```

Comment:
This variable name feels very inconsistent with all the other names (`Component`, ...). This also conflates tags and options, which I think makes the ext package harder to grasp.